ERIC_FORM.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+  <form>
+    <label>First Name:</label>
+    <input type="text" id="first_name">
+
+    <br>
+
+    <label>Surname:</label>
+  <input type="text" name="" id="Surname">
```

Comment:
No need for the "name" attribute.  You can remove it, especially if it is empty.